schemas/com.snowplowanalytics.snowplow/consent_preferences/jsonschema/1-0-0

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for consent preferences selection event",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "consent_preferences",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "eventType": {
```

Comment:
Of all the schemas in this PR, this is the only one to use cameCase instead of snake_case for field names.  Is this deliberate?

Reply:
These were two separate projects with different casing as it seems, not a deliberate decision.
Do we have a standard that every schema should follow moving forwards ?

Reply:
From Data Processing point of view, we do not have a standard and we allow anything.

But I wonder if your team wants to conform to a standard that you set for yourselves?

If you pick camel case, then events will look different between different warehouses.  For example, the field `basisForProcessing` -- in Snowflake this will look like `basisForProcessing` whereas in Redshift/Databricks/BigQuery the name will get converted to `basis_for_processing`.  Each warehouse will handle it just fine, but you end up with an inconsistency across different warehouses, which might be harder for you to manage.

If you pick `basis_for_processing` as the name of the original field then you don't have any inconsistency across warehouses.

I don't feel strongly about this, because it doesn't affect me or my team!!!  I'm just offering you this opportunity to think about whether your team needs a convention to stick to.

Reply:
For what it's worth, the `normalize` package does translate from camel to snake but it pulls the original case from the schema in the first place, so it shouldn't break if they were set as snake in the schema. However from what I've seen we do tend to use camel in most schemas I looked at.